sleap/gui/commands.py

```diff
@@ -2119,12 +2119,23 @@ def do_action(cls, context: CommandContext, params: dict):
             "Generating list of suggested frames... " "This may take a few minutes.",
             context.app,
         )
+        if len(context.labels.videos) == 0:
+            print("Error: no videos to generate suggestions for")
+        else:
```

Comment:
Lets get rid of this nested `if` `else` by adding a `return` in this `if` or by using an `elif` statement below. Whatever you prefer.